umake/ui/cli/__init__.py

```diff
@@ -157,6 +158,65 @@ def mangle_args_for_default_framework(args):
     return result_args
 
 
+def get_frameworks_list(args):
```

Comment:
`get_frameworks_list_output()` would finally make sense here (as we are going to give a string ready to be printed)